pyproject.toml

```diff
@@ -2,6 +2,17 @@
 requires = ["setuptools"]
 build-backend = "setuptools.build_meta"
 
+[project]
+name = "ImageDePHI"
+version = "0.1.0"
```

Comment:
`version` is a required field, so I'm arbitrarily choosing this number to start.